<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="API_AutoFramework"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.121">
    <properties></properties>

            <testcase
                name="getAPITest"
                time="0.121"
                classname="com.qa.API_F02.tests.GetAPITest_01_CG">

            <failure type="org.apache.http.client.ClientProtocolException" message="(null)">
                <![CDATA[org.apache.http.client.ClientProtocolException
                org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:187)org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)com.qa.API_F02.restclient.RestClient_get_01_CG.get(RestClient_get_01_CG.java:23)com.qa.API_F02.tests.GetAPITest_01_CG.getAPITest(GetAPITest_01_CG.java:37)sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)java.lang.reflect.Method.invoke(Method.java:498)org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)java.util.ArrayList.forEach(ArrayList.java:1257)org.testng.TestRunner.privateRun(TestRunner.java:764)org.testng.TestRunner.run(TestRunner.java:585)org.testng.SuiteRunner.runTest(SuiteRunner.java:384)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)org.testng.SuiteRunner.run(SuiteRunner.java:286)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)org.testng.TestNG.runSuitesLocally(TestNG.java:1140)org.testng.TestNG.runSuites(TestNG.java:1069)org.testng.TestNG.run(TestNG.java:1037)org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
                    Caused by: org.apache.http.ProtocolException: Target host is not specified
                    org.apache.http.impl.conn.DefaultRoutePlanner.determineRoute(DefaultRoutePlanner.java:71)org.apache.http.impl.client.InternalHttpClient.determineRoute(InternalHttpClient.java:125)org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)com.qa.API_F02.restclient.RestClient_get_01_CG.get(RestClient_get_01_CG.java:23)com.qa.API_F02.tests.GetAPITest_01_CG.getAPITest(GetAPITest_01_CG.java:37)sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)java.lang.reflect.Method.invoke(Method.java:498)org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)java.util.ArrayList.forEach(ArrayList.java:1257)org.testng.TestRunner.privateRun(TestRunner.java:764)org.testng.TestRunner.run(TestRunner.java:585)org.testng.SuiteRunner.runTest(SuiteRunner.java:384)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)org.testng.SuiteRunner.run(SuiteRunner.java:286)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)org.testng.TestNG.runSuitesLocally(TestNG.java:1140)org.testng.TestNG.runSuites(TestNG.java:1069)org.testng.TestNG.run(TestNG.java:1037)org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
                ]]>
            </failure>
    </testcase>


</testsuite>
